pyproject.toml

```diff
@@ -29,6 +29,9 @@ dependencies = [
   "frequenz-api-common >= 0.5.0, < 0.6.0",
   "googleapis-common-protos >= 1.56.2, < 2",
   "grpcio >= 1.51.1, < 2",
+  "frequenz-client-base @ git+https://github.com/frequenz-floss/frequenz-client-base-python@v0.x.x",
+  "frequenz-channels == 1.0.0b2",
```

Comment:
Do we need this high version? It complains when combined with the SDK

Reply:
@noah-kreutzer-frequenz Can you change this line to https://github.com/frequenz-floss/frequenz-sdk-python/blob/504b86f0e19fadd9afb4342a9468b59c26d84ca1/pyproject.toml#L33

Reply:
this is done.